### ISSUE TITLE
Quantum Incursions helper!

### ISSUE THREAD

**Issue by rendekp** [mainIine/foe-helfer-extension#2774]
Hi!
Is there any chance that you make something similar like the St. Patrick windows for the Quantum? 
I already found some useful city collection guide, but it hard to follow every time. The St. Patrick's strategy reminder was really useful!!

**Comment by roolop** [comment]
I wish there was something, even if it was just basic information.
Help on how to balance the buildings in order to have full happiness would be very positive.

**Comment by outoftheline** [comment]
as there are plenty of guides out there and you're doing the same thing every two weeks, we do not think this is necessary.
